src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte

```diff
@@ -83,6 +83,19 @@
 	{/each}
 </ul>
 
+<a
```

Comment:
In a follow-up PR, can you extract this in a specific component. `DappsExplorerSignedIn` starts to be big and this feautre can be scoped.